django/thunderstore/repository/api/v1/serializers.py

```diff
@@ -12,9 +12,16 @@ class PackageVersionSerializer(ModelSerializer):
     dependencies = SerializerMethodField()
 
     def get_download_url(self, instance):
-        url = instance.download_url
         if "request" in self.context:
-            url = self.context["request"].build_absolute_uri(instance.download_url)
+            url = self.context["request"].build_absolute_uri(
+                instance.download_url(
+                    self.context["view"].kwargs["community_identifier"]
```

Comment:
Isn't the community identifier available directly in the context, do we need to go through the view?

Reply:
Yes it is, no idea why how I didn't realize this.